<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grabación de voz</string>
  </property>
  <widget class="QLabel" name="readLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Lee el texto a continuación tras pulsar el botón &quot;Grabar voz&quot;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="recordButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Grabar voz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Continuar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
